src/runtime/link-accounts-flow-test.js

```diff
@@ -24,11 +24,13 @@ import {
   LinkCompleteFlow,
   LinkbackFlow,
   LinkSaveFlow,
+  LoginPromptFlow,
```

Comment:
It should be in its own module: this feature is pretty far away from linking.

Reply:
done